setup.py

```diff
@@ -43,6 +43,7 @@
         'test': [
             'flake8<6',
             'lxml',
+            'defusedxml',
```

Comment:
I'm not seeing this in use?

Reply:
No this isn't directly used, but it is being used by a sphinx testing plugin, and its requirements clearly aren't being set correctly, and some of the tests in matrix fail if it isn't present. Have added comment to make this clear.